tests/sampleTests/sampletest.spec.ts

```diff
@@ -1,19 +1,14 @@
-import { test } from '../testService';
-
-// Uncommmnet to continue in a headed browser
-// test.use({
-//     headless: false,
-//     viewport: { width: 1000, height: 1000 },
-// });
+import { test } from '../';
 
 test.describe('Sample Test Group', () => {
+    test('Bigpay Test Payment Provider is working', async ({paymentStep, page}) => {
+        await paymentStep.goto({ storeURL: 'https://my-dev-store-745516528.store.bcdev', harName: 'sample ' });
 
-    test('Bigpay Test Payment Provider is working', async ({testService, page}) => {
-
-        await testService.gotoPaymentStep({ storeURL: 'https://my-dev-store-745516528.store.bcdev', HAR: 'sample' });
-
-        if (testService.isReplay) {
-            await page.route('/checkout/payment/hosted-field?*', route => route.fulfill( {status: 200, path: './tests/sampleTests/_support/hostedField.html' } ));
+        if (paymentStep.isReplay) {
+            await paymentStep.playwright.serveFile({
+                routePath: '/checkout/payment/hosted-field?*',
+                filePath: './tests/sampleTests/support/hostedField.ejs',
```

Comment:
As stated in [EJS repo](https://github.com/mde/ejs/issues/281), EJS always looks for the file relative to `process.cwd()`, the directory where the Node.js process was started.  So we need to provide a template path like this or use `path.join(__dirname, RelativePath)` here.